ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -125,6 +126,12 @@ EOF
     sysctl -w net.ipv4.ip_forward=1
     sysctl -w net.ipv4.conf.${TUN}.forwarding=1
 
+    # tcp_mtu_probing controls TCP packetization-layer path MTU discovery
+    # Value: 1 - Disabled by default, enabled when an ICMP black hole detected
+    sysctl -w net.ipv4.tcp_mtu_probing=1
+    # Recommended value as per RFC#4821
+    sysctl -w net.ipv4.tcp_base_mss=1024
+
```

Comment:
Do we want to remove the option to configure this via the
DOCKER_NETWORK_SETTINGS envar? If that envar is set these settings won't
take effect and that could be confusing, if we do that then we'd want to
make sure that decide whether or not to include the bridge name as well.

On Mon, Aug 10, 2015 at 7:14 PM, Ravi Sankar Penta <notifications@github.com
> wrote:

> In ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh
> <https://github.com/openshift/openshift-sdn/pull/123#discussion_r36696008>
> :
>
> > @@ -125,6 +126,12 @@ EOF
> >      sysctl -w net.ipv4.ip_forward=1
> >      sysctl -w net.ipv4.conf.${TUN}.forwarding=1
> >
> > +    # tcp_mtu_probing controls TCP packetization-layer path MTU discovery
> > +    # Value: 1 - Disabled by default, enabled when an ICMP black hole detected
> > +    sysctl -w net.ipv4.tcp_mtu_probing=1
> > +    # Recommended value as per RFC#4821
> > +    sysctl -w net.ipv4.tcp_base_mss=1024
> > +
>
> These flags will help in case of black hole.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-sdn/pull/123/files#r36696008>.
>